includes/class-sensei-admin.php

```diff
@@ -444,13 +444,32 @@ public function register_scripts( $hook ) {
 		// Event logging.
 		Sensei()->assets->enqueue( 'sensei-event-logging', 'js/admin/event-logging.js', [ 'jquery' ], true );
 
+		// Add global admin data.
+		$this->localize_admin_data();
+
 		if ( $this->has_custom_navigation( $screen ) ) {
 			Sensei()->assets->enqueue( 'sensei-admin-custom-navigation', 'js/admin/custom-navigation.js', [], true );
 		}
 
 		wp_localize_script( 'sensei-event-logging', 'sensei_event_logging', [ 'enabled' => Sensei_Usage_Tracking::get_instance()->get_tracking_enabled() ] );
 	}
 
+	/**
+	 * Localize admin data that should be available on all admin pages.
+	 *
+	 * This method adds global data to the admin scripts, such as the upsell URL.
+	 *
+	 * @since 4.24.0
+	 * @access private
```

Comment:
These can be removed because it's a private method.

As standard, we use `@access`/`@internal` and `@since` on public methods. Some more info can be found here: p6rkRX-4Og-p2.

Specifically for `@since`, we use a special tag to indicate the next released version, `$$next-version$$`, which is replaced during the release process. More here: p6rkRX-67G-p2

Reply:
Thanks for the feedback. I've removed it. 👍